cernopendata_client/searcher.py

```diff
@@ -244,3 +251,24 @@ def get_file_info_remote(server, recid, protocol=None, filtered_files=None):
                 }
             )
     return file_info_remote
+
+
+def get_list_directory(path):
```

Comment:
Works well, but we need two things:

- add `recursive=True/False` option, and expose it in CLI via `-R, --recursive` analogously to the `ls` command

- launch the command as a separate process or thread, and kill it after ~30 seconds or so.  This could be configurable via `--timeout` option that would have default value defined in `config.py` to be say 60 seconds, as I mentioned in the ticket description.